pymc/tests/test_sampling.py

```diff
@@ -103,8 +103,8 @@ def test_sample(self):
     def test_sample_init(self):
         with self.model:
             for init in (
-                "advi",
-                "advi_map",
+                # "advi",
```

Comment:
Why are these commented out?

Reply:
they are still not working in V4

Reply:
That's a bug then as VI should work again.

Reply:
ohh you are right it seems the bug was introduced here https://github.com/pymc-devs/pymc/issues/4819 and is failing because we are doing https://github.com/pymc-devs/pymc/blob/0985bd47092107a004cc3bd7d93a210669b037fe/pymc/sampling.py#L2353 I will fix it after lunch :-)